src/schema/schema_worker.c

```diff
@@ -68,29 +68,86 @@ __schema_tiered_worker(WT_SESSION_IMPL *session, const char *uri,
 
 /*
  * __schema_layered_worker_verify --
- *     Run a schema worker operation (which is verification) on the stable table; ingest is not
- *     supported currently.
+ *     Run a schema worker operation (which is verification) on the layered table.
  *
- * FIXME-WT-15047: Implement ingest table verification (and update the function description)
+ * FIXME-WT-15047: Implement ingest table verification on followers.
  */
 static int
 __schema_layered_worker_verify(WT_SESSION_IMPL *session, const char *uri,
   int (*file_func)(WT_SESSION_IMPL *, const char *[]),
   int (*name_func)(WT_SESSION_IMPL *, const char *, bool *), const char *cfg[], uint32_t open_flags)
 {
+    WT_CONNECTION_IMPL *conn;
     WT_DECL_RET;
+    int ingest_ret, stable_ret;
+    const char *ingest_uri, *stable_uri;
+
+    conn = S2C(session);
+    ingest_ret = stable_ret = 0;
 
     WT_RET(__wt_session_get_dhandle(session, uri, NULL, NULL, open_flags));
     WT_LAYERED_TABLE *layered = (WT_LAYERED_TABLE *)session->dhandle;
-    const char *stable_uri = layered->stable_uri;
 
+    ingest_uri = layered->ingest_uri;
+    stable_uri = layered->stable_uri;
+
+    /*
+     * FIXME-WT-15413 - Verify assumes the stable table always exists. However, on followers that
+     * have not yet picked up their first checkpoint, the stable constituent will be missing. We
+     * should handle this transient state by skipping stable verification with a clear message
+     * instead of failing with ENOENT.
```

Comment:
I would propose providing a less detailed advice here since I am not sure whether we should print a message in case of skipping the stable table verification.


```suggestion
     * should handle this transient state by skipping stable verification
     * instead of failing with ENOENT.
```